test/units/testsuite-20.sh

```diff
@@ -152,7 +152,7 @@ runas() {
     declare userid=$1
     shift
     # shellcheck disable=SC2016
```

Comment:
I guess now the shellcheck can be enabled. Please drop the comment.

Reply:
ok, done